starknet_py/tests/e2e/docs/code_examples/test_full_node_client.py

```diff
@@ -14,6 +15,18 @@ def test_init():
     # docs-end: init
 
 
+@pytest.mark.asyncio
+async def test_init_with_custom_client_session():
+    # docs-start: init
+    # or with custom client session
+    session = ClientSession()
+    client = FullNodeClient(node_url="https://your.node.url", session=session)
+    # perform operations...
+    # close the session
+    await session.close()
+    # docs-end: init
+
+
```

Comment:
We could also change `test_init()` to be async, and put the example with custom client session creation into it.

Reply:
It is fine as it is now.